addons/portal/models/ir_qweb.py

```diff
@@ -15,9 +15,7 @@ def _prepare_frontend_environment(self, values):
         irQweb = super()._prepare_frontend_environment(values)
         values.update(
             is_html_empty=is_html_empty,
-            frontend_languages=lazy(lambda: [
-                lang for lang in irQweb.env['res.lang'].get_frontend_langs()
-            ])
```

Comment:
Since the method is ormcached, I don't think it will cause any performance issue right?

If the request is not for a website, it uses the ormcached result of `_get_active_by('code')` which is used for every http request and really unlikely to be invalidated.
If the request is for a webstie, I think the method (get_frontend_langs ) has been called in the http stack while checking the env.lang so the result should be cached also.

Reply:
`get_frontend_langs` was already cached before and still receive a `lazy` addition, I am not sure 100% why
Not something I would change for no good reason but as you prefer
Maybe @Gorash can tell if it's really something to not remove